R/derived-bmi.R

```diff
@@ -0,0 +1,72 @@
+#' Custom ifelse function that evaluates missing (NA) values
+#' 
+#' If the logical argument compares to a value that is NA, it will set `x` to `FALSE`  
+#' 
+#' @param x A logical argument
+#' 
+#' @param a value if `x` is `TRUE`
+#' 
+#' @param b value if `x` is `FALSE`
+#' 
+#' @return None
+#' 
+#' @examples ifelse2((age<18),"child", "adult")
+#' 
+#' @export
+ifelse2 <- function(x, a, b) {
+  falseifNA <- function(x) {
+    ifelse(is.na(x), FALSE, x)
+  }
+  ifelse(falseifNA(x), a, b)
+}
+
+#' Body Mass Index (BMI) derived variable
```

Comment:
Should we indicate the variable that the function is deriving in the description?

e.g. #' Body Mass Index (BMI) derived variable. Use to derive BMI_derived.